docker/docker-compose.yml

```diff
@@ -9,11 +9,11 @@ services:
             - "5432"
         environment:
             POSTGRES_USER: postgres
-            POSTGRES_PASSWORD: postgres
+            POSTGRES_PASSWORD: ${PW_POSTGRES}
     oms-discounts:
         build:
-            context: ./$PATH_OMS_DISCOUNTS/oms-discounts
-            dockerfile: ./Dockerfile.dev
+            context: ./$PATH_OMS_DISCOUNTS/..
+            dockerfile: ./docker/oms-discounts/Dockerfile
         image: aegee/oms-discounts:dev
         volumes:
             - oms-discounts-media:/usr/app/media
```

Comment:
what media are there?

also one line below there is a "`shared`" volume, what is it for/shared with?